src/go/rpk/pkg/cli/redpanda/admin/brokers/list.go

```diff
@@ -14,7 +18,27 @@ func newListCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 		Use:     "list",
 		Aliases: []string{"ls"},
 		Short:   "List the brokers in your cluster",
-		Args:    cobra.ExactArgs(0),
+		Long: `List the brokers in your cluster.
+
+This command lists all brokers in the cluster whatever a broker is up or down,
+unless it is decommissioned.
+
+The output table carries the following columns:
+
+ID               Node ID, an exclusive identifier for a broker
+HOST             Internal RPC address
```

Comment:
Does users recognize what is a RPC address? Do we have another name that we can use here ? Like "host internal API address" or something.

Reply:
`RPC address` itself is common but I should have made it clearer on what it actually is. How about `Internal RPC address for communication between nodes`?

Reply:
`Internal RPC address for communication between brokers`

Reply:
Thank you.